web/src/main/java/org/jboss/cdi/tck/tests/extensions/lifecycle/processBeanAttributes/builtin/ProcessBeanAttributesObserver.java

```diff
@@ -33,26 +34,32 @@ public class ProcessBeanAttributesObserver implements Extension {
     private List<ProcessBeanAttributes<?>> observedBeanAttributes = new ArrayList<ProcessBeanAttributes<?>>();
 
     public void observeHttpServletRequestBeanAttributes(@Observes ProcessBeanAttributes<HttpServletRequest> event) {
-        observedBeanAttributes.add(event);
+        if (event.getBeanAttributes().getQualifiers().contains(Default.Literal.INSTANCE)) {
+            observedBeanAttributes.add(event);
+        }
     }
 
     @SuppressWarnings("rawtypes")
     public void observeEventBeanAttributes(@Observes ProcessBeanAttributes<Event> event) {
-        observedBeanAttributes.add(event);
-    }
+        if (event.getBeanAttributes().getQualifiers().contains(Default.Literal.INSTANCE)) {
+            observedBeanAttributes.add(event);
+        }    }
```

Comment:
Formatting :roll_eyes:

Reply:
Interesting, that's not what I saw in IDEA locally...
Formatted the whole file and force pushed, seems to be fine now.